### PR TITLE
docs(rustlang): Rust Hello World instructions

### PR DESCRIPTION
**Pull request by DBBrowne**
Please let me know if this is not correct, as that would mean that I did something wrong!

### CODE REVIEW

**Comment by coolaj86** [comment]
Thanks for the fix!

**Review by coolaj86 — APPROVED** [review]

File: rustlang/README.md [hunk 1]
```diff
@@ -39,7 +39,8 @@ cargo install ripgrep
 
 ```bash
 cargo new hello --bin
+cd hello
```
Comment (coolaj86) [inline rustlang/README.md:42]: If you'd like to improve this:

```bash
pushd ./hello/
```

`pushd` is what modern shells actually use when you type `cd`, and it works more reliably, even cross-platform (Windows too).

Comment (DBBrowne) [reply rustlang/README.md]: Sure, clear and explicit sounds better.

`cd` works on windows, in cmd and Powershell, however it doesn't seem to add the prior directory to whatever history pushd/popd uses, so popd does not work after a cd. Interesting.

**Review by coolaj86 — APPROVED** [review]

**Comment by coolaj86** [comment]
Thank you!

If you'd like me to buy you lunch, just drop your Đash address here and I'll send you a tip. :)

**Pull request merged**